Views/GICommitRewriterViewController.m

```diff
@@ -159,7 +159,7 @@ - (BOOL)startRewritingCommit:(GCHistoryCommit*)commit error:(NSError**)error {
   [self.repository notifyRepositoryChanged];
   
   _targetCommit = commit;
-  _titleTextField.stringValue = [NSString stringWithFormat:@"\"%@\" <%@>", _targetCommit.summary, _targetCommit.shortSHA1];
```

Comment:
For consistency with the rest of the source code, you actually need to use NSLocalizedString().